client/cgo/verification.go

```diff
@@ -55,13 +56,15 @@ func cgoVerifyVrf(pk unsafe.Pointer, pkSize C.int,
 }
 
 //export cgoVerifyHashChain
-func cgoVerifyHashChain(prevHash unsafe.Pointer, root unsafe.Pointer, size C.int) C.int {
-	if int(size) != crypto.HashSizeByte {
+func cgoVerifyHashChain(prevHash unsafe.Pointer, hashSize C.int, strSig unsafe.Pointer, sigSize C.int) C.int {
```

Comment:
I'd prefer to keep `prevHash` and rename `strSig` to `savedStrSig`. Fixed in 4fdc1088889c6ae07e4d02b67887fcf635ea8d11